.vscode/extensions.json

```diff
@@ -7,7 +7,8 @@
     "ms-python.python",
     "redhat.vscode-yaml",
     "seatonjiang.gitmoji-vscode",
-    "tamasfe.even-better-toml"
+    "tamasfe.even-better-toml",
+    "dprint.dprint"
```

Comment:
If I may pick a nit; It'd help appease my neuroticism if this were to be sorted alphabetically.